scripts/tags/exturl.js

```diff
@@ -4,8 +4,11 @@
 
 'use strict';
 
-var util = require('hexo-util');
+/*jshint camelcase: false */
+var util = require(hexo.base_dir + 'node_modules/hexo/node_modules/hexo-util');
```

Comment:
It's path to the `hexo-util`, on clean install all work.

1. If we will install clean hexo, in `node_modules` will not `hexo-ulil`.
2. If we will install `hexo-util` with cusom command, module will be in `node_modules`, not in `hexo` node-modules.

I don't undestand why Tommy do that, but there is 2 different path on same `hexo-util`.